lib/trino-filesystem-manager/src/main/java/io/trino/filesystem/manager/FileSystemConfig.java

```diff
@@ -20,9 +20,9 @@
 public class FileSystemConfig
 {
     private boolean hadoopEnabled = true;
```

Comment:
Whenever we switch .. this should be off by default

Reply:
Don't think so, unless we want `hdfs://` URLs not to work by default (which I don't think we want).

Reply:
Only if there are no side effects .. hdfs is more and more legacy so I think it might be okay to have it off and disabled. Might even be okay to have all of them off by default and require users to explicitly enable only what they need.